src/components/card.js

```diff
@@ -1,27 +1,22 @@
 import React from "react";
 import "./card.css";
 import { Card as BsCard } from "react-bootstrap/";
-import { SlLogin } from "react-icons/sl";
-
-function Card({ title, content, label }) {
-  const getArrowDirection = () => {
-    return content >= 150 ? "up" : "down";
-  };
 
+function Card({ title, content, label, isPositive, icon }) {
   const getArrowColor = () => {
-    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+    return isPositive ? "#009E0F" : "#CC0000";
   };
 
   return (
     <BsCard.Body className="cardArea">
-      <BsCard.Title className="cardTitle d-flex justify-content-between p-2">
+      <BsCard.Title className="cardTitle  p-2">
```

Comment:
consider using grid in cardTitle, avoid using custom CSS and use built in component from react-bootstrap, Grid will help you align your elements for the title and the icon. Icon will still be in the right side even if there is no title

https://react-bootstrap.netlify.app/docs/layout/grid